modules/music/api/music-routes.ts

```diff
@@ -8,6 +8,15 @@ interface Music {
     song: string
 }
 
+interface Album {
```

Comment:
Can you move this interface into a new folder under music called 'consts' please. :)